xc/device.ml

```diff
@@ -3337,8 +3337,7 @@ module Dm = struct
       let cancel = Cancel_utils.Vgpu domid in
       if not (Vgpu.is_running ~xs domid) then begin
         let pcis = List.map (fun x -> x.physical_pci_address) vgpus in
-        List.iter (fun x -> ignore(PCI.dequarantine xc x)) pcis;
```

Comment:
There is a another `dequarantine` call below. To avoid confusion, can we do all such calls in the same place? E.g. widen the map in `VM_start` to all PCI devices, and delete the `quarantine` call below?